cli/command_snapshot_create.go

```diff
@@ -20,6 +20,7 @@ import (
 const (
 	maxSnapshotDescriptionLength = 1024
 	timeFormat                   = "2006-01-02 15:04:05 MST"
+	numberOfPartsInTagString     = 2
```

Comment:
nit: that type of constant is very local and makes sense to define in the function or next to it, not at the top of the file. Perhaps even not have a constant but //nolint:gomnd because it's always going to be 2 anyway.